routes/web.php

```diff
@@ -1,37 +0,0 @@
-<?php
-
-/*
-|--------------------------------------------------------------------------
-| Web Routes
-|--------------------------------------------------------------------------
-|
-| Here is where you can register web routes for your application. These
-| routes are loaded by the RouteServiceProvider within a group which
-| contains the "web" middleware group. Now create something great!
-|
-*/
-
-Route::get('/blog/{post}/{slug}', 'BlogController@getSingle');
```

Comment:
a {post} and a {slug}? I think it should be either a post id or a slug.

Reply:
slug heb ik toegevoegd als titel van de post in de url, voor seo friendlyness? en omdat 2 titels hetzelfde kunnen zijn heb ik de postid eerst